api-specs/Gateway-EE/latest/kong-ee.yaml

```
@@ -5777,6 +5777,153 @@ info:
   version: 3.6.0
 openapi: 3.0.0
 paths:
+  /:
+    get:
+      summary: Get Kong's instance information
+      tags:
+        - Information
+      operationId: geInfo
+      description: |
+        Returns basic information about the Kong gateway instance, including the version, edition (enterprise or community), a tagline, and the unique node identifier.
```

```suggestion
        Returns detailed information about the Kong gateway instance, including the full Kong configuration, available and unavailable plugins, version, edition (enterprise or community), a tagline, the unique node identifier, and other metadata.
```
Something like that maybe